### DISCUSSION TITLE
(v2) Feature Request: Move Queue To a Different Speaker / Group

### DISCUSSION THREAD

**Discussion by RJ-Make** [music-assistant/support#1178] [Feature requests and ideas]
Currently the Queue stays attached to a speaker / group. It would be great to have a function that allows one to move the existing queue to another speaker or group.

Use case:  I was playing through a queue in my office and needed to move upstairs so I wanted to change the speaker to the upstairs speakers so I could continue on those speakers.  This is currently not possible.

**Comment by OzGav** [comment]
Duplicate of https://github.com/orgs/music-assistant/discussions/553
